packages/components/organisms/f-header/src/components/UserNavigationPanel.vue

```diff
@@ -6,7 +6,8 @@
         <li
             v-for="(link, index) in copy.navLinks"
             :key="index"
-            data-test-id="nav-links">
+            data-test-id="nav-links"
+            :class="$style['outer-link']">
```

Comment:
Just to fit with the naming that already exists:

```suggestion
            :class="$style['c-user-list-item']">
```